### ISSUE TITLE
Feature request: Only version specific package_sets

### ISSUE THREAD

**Issue by planthaber** [rock-core/autoproj#42]
For the versioning fuctionality:

I think it could be a very good feature to be able to version specific package sets and not only the complete version state.

For example in one of my current projects there are some works with very active development in their own package set, which i whould like to keep following on HEAD, but e.g. i don't want the head of rock.core and orocos.toolchain to be on head.

What do you think?

**Comment by doudou** [comment]
You should be able to pass package and package set names to the command line of versions/commit and tag. Did you try it ?

**Comment by planthaber** [comment]
No, asked Jakob if this is planned, he advised me to write this feature request.

**Comment by doudou** [comment]
>> Did you try it ?
> No

I assume that your answer means "yes you tried it" and "no it did not work". Right ?

**Comment by planthaber** [comment]
The answer means, I dindn't try, I asked Jakob whether the feature is included or not.

His aswer was it is not included and I should start a feature request, which i did here.
